OpenRA.Mods.Common/Activities/CaptureActor.cs

```diff
@@ -28,6 +28,18 @@ public CaptureActor(Actor self, Target target)
 			manager = self.Trait<CaptureManager>();
 		}
 
+		protected override void TickInner(Actor self, Target target, bool targetIsDeadOrHiddenActor)
+		{
+			if (enterActor != target.Actor)
+			{
+				enterActor = target.Actor;
+				enterCaptureManager = target.Actor.TraitOrDefault<CaptureManager>();
+			}
+
+			if (!enterCaptureManager.CanBeTargetedBy(enterActor, self, manager))
```

Comment:
We're using `TraitOrDefault`, so `enterCaptureManager` can be `null` here. If we're sure the trait exists use just `Trait` instead.